pyart/graph/radardisplay.py

```diff
@@ -1019,6 +1050,44 @@ def _label_axes_vpt(self, axis_labels, ax):
         else:
             ax.set_ylabel(y_label)
 
+    def _set_vpt_time_axis(self, ax, dForm=None, tz=None,
+                           date_MinTicker=None):
+        """ Set the x axis as a time formatted axis.
+
+        Parameters
+        ----------
+        ax : Matplotlib axis instance
+            Axis to plot. None will use the current axis.
+        dForm : str
```

Comment:
Oops, I must have forgotten to post the comment this was referring to after writing it.  

Py-ART uses lowercase_with_underscores for function arguments, so I think `time_form` and `date_minor_ticker` would be better names for `dForm` and `date_MinTicker`